### PR TITLE
Modify Export-StartLayout script documentation for Windows 11

### PR DESCRIPTION
**Pull request by danielaalpe**
## Why is this change being made
To update the documentation with guidance for using the script on Windows 11.

## What changed
New syntax and example were added.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by danielaalpe** [comment]
Hi @traya1, can you please take a look at this and merge it?

**Comment by traya1** [comment]
Hi @JohanFreelancer9,

Is this PR waiting on another approval on your team's end or is it ok to be sent to PubOps?

**Comment by JohanFreelancer9** [comment]
@traya1 - The PR is good to go on my end, thanks.

**Pull request merged**
